fe/fe-core/src/main/java/com/starrocks/system/SystemInfoService.java

```diff
@@ -263,6 +264,11 @@ public void dropBackend(String host, int heartbeatPort, boolean needCheckUnforce
         final Cluster cluster = GlobalStateMgr.getCurrentState().getCluster(droppedBackend.getOwnerClusterName());
         if (null != cluster) {
             cluster.removeBackend(droppedBackend.getId());
+            // remove worker
+            if (Config.integrate_starmgr) {
+                String starletHost = droppedBackend.getHost() + ":" + droppedBackend.getStarletPort();
```

Comment:
Starlet port may be not reported when FE has just started.

Reply:
What to do if remove worker fails?
Maybe we can try to remove worker first, and return error if remove worker fails.